docs/reference/changelog.md

```diff
@@ -2,12 +2,63 @@
 
 [[toc]]
 
+## n8n@0.74.0
+For a comprehensive list of changes, check out the [commits](https://github.com/n8n-io/n8n/compare/n8n@0.73.1...n8n@0.74.0) for this version.
+
+- New nodes
+    - Hacker News
+    - QuestDB
+    - Xero
+- Enhanced nodes
+    - Affinity Trigger
+    - HTTP Request
+    - Mailchimp
+    - MongoDB
+    - Pipedrive
+    - Postgres
+    - Uplead
+    - Webhook    
+- Webhook URLs are now handled independently of the workflow ID by `https://{hostname}/webhook/{path}` instead of the older `https://{hostname}/webhook/{workflow_id}/{node_name}/{path}`.
+
+
+## n8n@0.73.1
+For a comprehensive list of changes, check out the [commits](https://github.com/n8n-io/n8n/compare/n8n@0.73.0...n8n@0.73.1) for this version.
+
+- Enhanced nodes
+    - Microsoft SQL
+
+
+## n8n@0.73.0
+For a comprehensive list of changes, check out the [commits](https://github.com/n8n-io/n8n/compare/n8n@0.72.0...n8n@0.73.0) for this version.
+
+- New nodes
+    - CircleCI
+    - Salesforce
+    - Zoom
```

Comment:
Can you please make sure that the nodes are always alphabetically listed in the Changelog?